src/transformStateWithClones.js

```diff
@@ -7,7 +7,37 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [state];
```

Comment:
```suggestion
  const result = [];
```

why do we need add this to array and slice when return?